graphql_compiler/compiler/filters.py

```diff
@@ -113,13 +113,13 @@ def _represent_argument(directive_location, context, argument, inferred_type):
 
         return (expressions.Variable(argument, inferred_type), None)
     elif is_tagged_parameter(argument):
-        argument_context = context['tags'].get(argument_name, None)
```

Comment:
Thanks! Is anything else using `context['tags']`? If not, it's probably safe to delete.

Reply:
Nothing else. I did delete it.

Reply:
Let's make sure `context['tags']` is never set up in the first place, and confirm with a grep for `context['tags']`:
https://github.com/kensho-technologies/graphql-compiler/pull/351/files#diff-fffee83da4144bc70ab3578390d57761R827